package.json

```diff
@@ -31,6 +31,7 @@
     ]
   },
   "dependencies": {
+    "@storybook/addon-centered": "^3.4.3",
```

Comment:
Go ahead and remove the `^` so that we're fixed to exact versions on our dependencies. 
We don't want any surprises in our production versions.